### PR TITLE
84 sprint 3 edit naming and routing of project

### PR DESCRIPTION
**Pull request by ozzy1323232**

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -111,23 +111,34 @@ def session():
     set_session_form_select_options(form)
     return flask.render_template('session.html', form=form, perth_time=formatted_perth_time)
 
-#ADMIM - /admin/
+#ADMIM - /unitconfig /
+@app.route('/unitconfig', methods=['GET', 'POST'])
+@login_required
```
Comment (DeltaO3) [inline app/routes.py:116]: Still entirely accessible if not a facilitator by typing the route into the searchbar, should be backend validation of user type and redirect if so

File: app/routes.py [hunk 2]
```diff
@@ -111,23 +111,34 @@ def session():
...
+        return flask.redirect(flask.url_for('admin'))
+    
+    return flask.render_template('admin.html', form=form)
 
-# ADDUNIT - /addunit/ /admin/
+# ADDUNIT - /addunit/ /unit/
 @app.route('/addunit', methods=['GET', 'POST'])
 @login_required
 def addunit():
-    if current_user.userType != 1:
```
Comment (DeltaO3) [inline app/routes.py:129]: This should not be removed, can currently access as a facilitator

File: app/templates/navbar.html [hunk 3]
```diff
@@ -12,16 +12,31 @@
...
-					href="{{ url_for('session')}}">Config</a>
-				<!-- TODO: add jinja login conditional-->
-				<a class="nav-link {% if request.path == url_for('admin') %}active" aria-current="page{% endif %}"
-					href="{{ url_for('admin')}}">Admin</a>
+					aria-current="page {% endif %}" href="{{ url_for('home') }}">
+					Home
+				</a>
+
+				<!-- Session Config: Visible to admins (1), coordinators (2), and facilitators (3) -->
+				{% if current_user.userType == 1 or current_user.userType == 2 or current_user.userType == 3 %}
```
Comment (DeltaO3) [inline app/templates/navbar.html:22]: No reason to have an if statement that checks all possible types, will always evaluate to true.

**Review by DeltaO3 — CHANGES_REQUESTED** [review]
Some issues with no backend validation of user type, and a pointless if statement, otherwise looks fine.
It is worth noting that eventually in the future we will make log in page not display the navbar.

File: app/routes.py (hunk 2 above)
Comment (ozzy1323232) [reply app/routes.py:129]: done

**Review by DeltaO3 — APPROVED** [review]
Pretty sure this is good, I forget if unit coordinators can add units but i think they can, so its correct.

**Review by ipr17 — COMMENTED** [review]
Looks good to me as well, happy for this to be merged

**Pull request merged**